OpenRA.Mods.Common/Traits/Conditions/ExternalCondition.cs

```diff
@@ -104,7 +104,7 @@ public int GrantCondition(Actor self, object source, int duration = 0, int remai
 				if (Info.SourceCap > 0)
 				{
 					var timedCount = timedTokens.Count(t => t.Source == source);
-					if ((permanent != null ? permanent.Count + timedCount : timedCount) >= Info.SourceCap)
+					if ((permanent?.Count + timedCount ?? timedCount) >= Info.SourceCap)
```

Comment:
This isn't right.

Reply:
Isn't it? This was one of the automated fixes by Rider.